doc/how-to/setup.md

```diff
@@ -5,10 +5,24 @@
 
 ## Project dependencies
 
-- NodeJS
-- pnpm: `sudo corepack enable`
+* Version control system [Git](https://git-scm.com)
+	* On MacOS: `brew install git`
+* [NodeJS LTS](https://nodejs.org) (actual in use is 20.17.0)
+	* On MacOS: `brew install node@20`
+* Package manager [pnpm](https://pnpm.io/)
+	* On MacOS: `brew install pnpm`
+	* Enable corepack afterwards : `sudo corepack enable`
+* If you are using VSCode IDE:
+	* [Svelte extension](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode)
 
 ## Core's dependency
 
-- chrome or chromium installed and available in terminal  
-  `brew install chromium`
\ No newline at end of file
+* Chrome or [Chromium](https://www.chromium.org) installed and available in terminal  
+	* On MacOS: `brew install chromium`
+
+## Miscellaneous
+
+* Team member of the github [repo](https://github.com/sprinteins/oscd-plugins)
+* Team member of the github [TransnetBW Project](https://github.com/orgs/sprinteins/projects/7)
+* Figma Read-Access
+* Mural Read-Access
```

Comment:
These are internal tools. If external developers want to contribute we will find another way. I don't think we need Mural for development, everything that is necessary for the development should be part of the issue.